### DISCUSSION TITLE
fetching the VMS or motion detection view in  developer tool called QT application

### DISCUSSION THREAD

**Discussion by Pranay438** [AxisCommunications/discussions#178] [VAPIX]
camera is streaming normal along with VMD application in web browser but its not fetching the VMS or motion detection view in developer tool called QT application.we tried passing username and password in url from our application but it is not allowing me to login, it is throwing "Error in loading" we need to access the motion detection view in our application

**Comment by mattias-kindborg-at-work** [comment]
Hi @Pranay438. Which is this application you are referring to? Can you link to it on [axis.com](https://www.axis.com/)?

**Reply by TedduPranay** [reply]
Qt application  , we are building a desktop application using qt creator ide where we need to embed motion detection tab from axis web page into our desktop application.to access vmd we need to login we tried to login through url by specifying username and password in url.but it is throwing 'error in loading' .our aim is to access vmd in our desktop application.

**Reply by TedduPranay** [reply]
Awaiting for the reply
